docs/sources/administration/change-home-dashboard.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Change the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "change-home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# Change the default home dashboard
+
+You can change the default homepage of Grafana in two different ways: change Grafana preferences or change the Grafana configuration.
```

Comment:
We call it home dashboard or default dashboard everwhere else, I think we should use either over homepage to stay consistent.